samples/drivers/led_ws2812/README.rst

```diff
@@ -74,6 +74,9 @@ For example devicetree configurations for each compatible, see
 Some boards are already supported out of the box; see the :file:`boards`
 directory for this sample for details.
 
+The sample updates the LED strip periodically. The update frequency can be
+modified by changing the `SAMPLE_LED_UPDATE_DELAY`.
```

Comment:
```suggestion
modified by changing the ``SAMPLE_LED_UPDATE_DELAY``.
```

Reply:
Added `:kconfig:option:`, which has been used elsewhere in the file.
What do you use to edit Kconfig files to help with syntax?

Reply:
> Added `:kconfig:option:`, which has been used elsewhere in the file. 

Great call :)

> What do you use to edit Kconfig files to help with syntax?

I am assuming you might mean rst files, not Kconfig? 
If the former, I didn't really find a good solution so far https://marketplace.visualstudio.com/items?itemName=lextudio.restructuredtext gets some of the work done but it's pretty buggy (the kind of buggy where sometimes it becomes impossible to use the SPACE key anymore 😅)
I don't touch Kconfigs often so don't really have any editor support there

Reply:
Yes I meant rst files :)
Alright I guess I'll stick with `nvim` for now then.